posthog/models/web_preaggregated/sql.py

```diff
@@ -1,17 +1,13 @@
-from django.conf import settings
```

Comment:
Those are not related. I tried to send them separately here https://github.com/PostHog/posthog/pull/35002 but the CI is hanging